avalon/tools/cbloader/app.py

```diff
@@ -129,6 +150,9 @@ def _refresh(self):
         assets_model.refresh()
         assets_model.setFocus()
 
+        family_filter_widget = self.data["widgets"]["filter"]
```

Comment:
Aside from forcing a refresh, why do we need to refresh the `family_filter_widget` when we're changing assets? Does changing to another asset actually result in different contents?